site/content/docs/main/config/access-logging.md

```diff
@@ -76,6 +76,10 @@ Note that the `DYNAMIC_METADATA` and `FILTER_STATE` Envoy logging operators are
 
 See the [example config file][6] to see this used in context.
 
+#### Omitting Logs with Empty Values
+
+Contour now automatically omits empty fields in Envoy JSON access logs, enhancing clarity and delivering more concise and relevant log outputs by default.
```

Comment:
Looks good to me too, but I have one more super small wish: could you remove "now" from the documentation
```suggestion
Contour automatically omits empty fields in Envoy JSON access logs, enhancing clarity and delivering more concise and relevant log outputs by default.
```
it would be "age" bit better :smile: